tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -94,11 +94,12 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
                 // these are executed with exec, so we could check for that, but the
                 // below check is more conservative, so leaving at that
                 process_command_line.find(WStr("testhost")) == WSTRING::npos &&
+                process_command_line.find(WStr("exec")) == WSTRING::npos &&
```

Comment:
Based on the comment above, I'd assume we _don't_ need to add `exec` here? What command line call are we trying to accommodate now? 🤔

Reply:
I'll remove the comments, because starting xUnit v3 they are not using `testhost` to run the tests, they create an standalone binary, so we need to instrument that. The same will happen with TUnit I guess...

Reply:
removed the comment in https://github.com/DataDog/dd-trace-dotnet/pull/6573/commits/0bb1ed8bf288df45f34491c03669193dd5f59b8b